tests/lax_control_flow_test.py

```diff
@@ -2044,34 +2094,58 @@ def sqrt_cubed(x, tangent_solve=scalar_solve):
 
     inputs = jnp.array([4.0, 5.0])
     results = jax.vmap(sqrt_cubed)(inputs)
-    self.assertAllClose(results, inputs ** 1.5, check_dtypes=False)
+    self.assertAllClose(results, inputs ** 1.5, check_dtypes=False, atol=1e-6, rtol=1e-6)
 
     results = jax.jit(sqrt_cubed)(5.0)
     self.assertAllClose(
         results, 5.0**1.5, check_dtypes=False, rtol={np.float64: 1e-7})
 
+  @parameterized.named_parameters([
+    {"testcase_name": "vector_solve", "solve_method": vector_solve, "modes": ["fwd", "rev"]},
+    {"testcase_name": "newton_raphson", "solve_method": newton_raphson, "modes": ["fwd"]},
```

Comment:
The reason why this is failing is because the unit test is checking second order gradients (e.g., VJP  ​of JVP), and `newton_raphson` itself is not differentiable. So this is expected behavior -- custom_root can only support higher order auto-diff is the supplied linear solver is also differentiable.

This could be fixed in a couple of ways:
1. make `newton_raphson` itself differentiable with a custom gradient rule via `custom_root` (this feels a little overly complex)
2. only check first order gradients for this test (`order=1` in `check_grads`).
3. drop this casse entirely, because Newton-Raphson is already being tested sufficiently elsewhere.

My preferred option would be (2) or (3), but I would lean slightly towards (3)